src/commissaire_service/service/__init__.py

```diff
@@ -250,9 +246,7 @@ def _wrap_on_message(self, body, message):
                 message.properties['reply_to']))
             response_queue = self.connection.SimpleQueue(
                 message.properties['reply_to'])
-            response_queue.put({
-                'result': json.dumps(result),
-            })
+            response_queue.put(json.dumps(response))
```

Comment:
Note: This is correct, but I noticed by doing this the ``simple.wordy_add`` example breaks as it is not expecting a string, but a dictionary. In ``commissaire/src/commissaire/bus/__init__.py`` line 102 we will need to add a ``json.load`` to ensure we have a dictionary again.